app/code/Magento/Sales/Model/Order/Invoice.php

```diff
@@ -142,7 +142,7 @@ public function __construct(
         \Magento\Framework\Api\ExtensionAttributesFactory $extensionFactory,
         AttributeValueFactory $customAttributeFactory,
         \Magento\Sales\Model\Order\Invoice\Config $invoiceConfig,
-        \Magento\Sales\Model\OrderFactory $orderFactory,
```

Comment:
It's not allowed to change constructor signature, especially for classes marked with `@api` annotation.
More information is available here: http://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/index.html